packages/compass-sidebar/src/components/non-genuine-marker.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import {
+  Badge,
+  BadgeVariant,
+  Icon,
+  css,
+  spacing,
+  useFocusRing,
+} from '@mongodb-js/compass-components';
+
+const nonGenuineMarkerContainer = css({
+  padding: `0 ${spacing[2]}px ${spacing[2]}px`,
```

Comment:
```suggestion
  padding: `0 ${spacing[3]}px ${spacing[3]}px`,
```

Let's align this with the fle marker?

I was also looking at it a bit, how do you feel about a little bit of margin on the right of the badges?
Before:
<img width="280" alt="Screen Shot 2022-08-30 at 7 46 47 AM" src="https://user-images.githubusercontent.com/1791149/187429560-04fd5664-0bff-400d-8280-3e7b077cc5a4.png">

After:
<img width="275" alt="Screen Shot 2022-08-30 at 7 50 45 AM" src="https://user-images.githubusercontent.com/1791149/187429545-26dcc1e4-e470-4e24-b6f5-934873152c9f.png">

```ts

const iconStyles = css({
  marginRight: spacing[1]
});
```

Reply:
oops. meant to go back and fix it up :)